ajo/personalization-client-side/README.md

```diff
@@ -0,0 +1,112 @@
+# Client-side Personalization
+
+## Overview
+
+This sample demonstrates using Adobe Experience Platform to get personalization content from Adobe Journey Optimizer. The web page changes based on the personalization content returned.
+
+This sample uses the [Adobe Experience Platform Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) to get personalization content and to render it entirely client-side.
+
+Here is what the page looks like before and after personalization content is rendered.
+
+| without personalization                                        | with personalization                                                 |
+|----------------------------------------------------------------|----------------------------------------------------------------------|
+| <img src="../../.assets/plain.png" alt="drawing" width="800"/> | <img src="../../.assets/with-offers.png" alt="drawing" width="800"/> |
+
+Please review the [summary of personalization content](../../ajo/TargetActivities.md) for this sample.
+
+Keep in mind that personalization content can come from any Adobe source such as [Journey Optimizer](https://experienceleague.adobe.com/docs/journey-optimizer/using/ajo-home.html?lang=en) or [Target](https://experienceleague.adobe.com/docs/target.html). Target is the source of personalization content used in this sample, but the implementation described here will work for all personalization solutions Adobe offers.
+
+## Running the sample
+
+<small>Prerequisite: [install node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).</small>
+
+To run this sample:
+
+1. [Setup local SSL certificates for https](../../LocalSSLCertificateSetup.md).
+2. Clone the repository to your local machine.
+3. Open a terminal and change directory to this sample's folder.
+4. Run `npm install`
+5. Run `npm start`
+6. Open a web browser to [https://localhost](https://localhost)
+
+## How it works
+
+1. [Alloy](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) is included on the page.
+2. The `sendEvent` command is used to fetch personalization content.
+
+```javascript
+alloy("sendEvent", {
+  renderDecisions: true,
+  decisionScopes: ["sample-json-offer"],
+}).then(applyPersonalization("sample-json-offer"));
+```
+
+3. Alloy renders page load Visual Experience Composer (VEC) offers automatically because the `renderDecisions` flag is set to true.
+4. Form-based JSON offers are manually applied by the sample implementation code (in the [`applyPersonalization`](public/script.js) method) to update the DOM based on the offer.
```

Comment:
need to change this from Form-based JSON offers to "AJO Code-based experience JSON offers"

Reply:
Done